cmd/main.go

```diff
@@ -175,8 +182,21 @@ func main() {
 		}
 
 		// Create a new backend instance.
-		backend, err := backends.NewSQLBackend(conn, opt, sLog)
-		if err != nil {
+		var backend backends.ResultBackend
+		var errI error
+
+		switch cfg.Type {
+		case dbMySQL:
+		case dbPostgres:
+		case dbTypeOracle:
+			backend, errI = backends.NewSQLBackend(conn, opt, sLog)
+
+		case dbTypeCsvq:
+			backend, errI = backends.NewCsvqBackend(conn, opt, sLog)
```

Comment:
Because with this driver... the code change a few... decide implement new backend .. this aproach (not use ifs with typo) could help later to load this code dinamically